README.md

```diff
@@ -8,71 +8,117 @@
   <em>CLI to work with your AsyncAPI files. Currently supports validation, but it is under development for more features.</em>
 </p>
 
-### Install
-```bash
-$ npm install --global @asyncapi/cli
+<p align="center">
+<a href="https://github.com/asyncapi/cli/blob/master/LICENSE"><img alt="GitHub license" src="https://img.shields.io/github/license/asyncapi/cli"></a>
+<a href="https://github.com/asyncapi/cli/actions/workflows/if-nodejs-pr-testing.yml">
+<img src="https://github.com/asyncapi/cli/actions/workflows/if-nodejs-pr-testing.yml/badge.svg" alt="PR testing - if Node project"  />
+</a>
+<a href="https://www.npmjs.com/package/@asyncapi/cli">
+<img alt="npm" src="https://img.shields.io/npm/dw/@asyncapi/cli">
+</a>
+
+</p>
+
+## Table of contents
+
+1. [Getting Started](#getting-started)
+    - [Installation](#installation)
+2. [Usage](#usage)
+3. [API Reference](#api-reference)
+    - [`validate`](#validate)
+    - [`context`](#context)
+4. [Contributing](#contributing)
+
+## Getting Started
+To run `@asyncapi/cli`, ensure that you have NodeJs >=v10. [Install NodeJs via package manager](https://nodejs.org/en/download/package-manager/).
```

Comment:
should be `Node.js`, and make sure it is consistent everywhere

Reply:
I wonder if the Requirements section sounds and looks better, like in here -> https://github.com/asyncapi/modelina#requirements